### PR TITLE
split and update CHANGELOG

### PR DESCRIPTION
**Pull request by Geometer1729**
Changelog stuff I forgot in #88

### CODE REVIEW

**Comment by AriFordsham** [comment]
You didn't forget them, I did

File: psm/CHANGELOG [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the [Haskell Package Versioning Policy](https://pvp.haskell.org/).
 
+## [0.5.2.1] - 2023-02-07
+
+### Removed
```
Comment (AriFordsham) [inline psm/CHANGELOG:10]: Unnecessary documenting internals is an antipattern

**Review by AriFordsham — APPROVED** [review]

File: psm/CHANGELOG (hunk 1 above)
Comment (Geometer1729) [reply psm/CHANGELOG:10]: Fair enough, I removed it and added info about the change in downstream nix imports.

**Review by AriFordsham — APPROVED** [review]
Nice catch.

**Pull request merged**